docs/guide/snaps-development-guide.md

```diff
@@ -1,14 +1,10 @@
 # Development Guide
 
-::: tip Only Available in MetaMask Flask
-[Snaps](./snaps.html) is only available in [MetaMask Flask](https://metamask.io/flask).
+::: tip Developer Preview Software 
+Snaps is pre-release software. To try Snaps, install [MetaMask Flask](https://metamask.io/flask).
```

Comment:
Any reason to remove the backlink to snaps.html?

Reply:
This was a mistake. I have added the backlink again.